contracts/Zauction.sol

```diff
@@ -332,7 +432,8 @@ contract ZAuction is Initializable, OwnableUpgradeable {
     uint256 bid,
     address owner,
     uint256 topLevelId,
-    uint256 tokenId
+    uint256 tokenId,
+    IERC20 paymentToken
   ) internal {
     address topLevelOwner = hub.ownerOf(topLevelId);
     uint256 topLevelFee = calculateTopLevelDomainFee(topLevelId, bid);
```

Comment:
Now that we're supporting multiple ERC-20 tokens, will there be any changes needed in our topLevelDomainFee calculation logic?

Reply:
resolved during standup, no changes for top level domain calculation formula specifically but may need to account for the `decimals` property of an ERC20 token in other places.